DEVELOPMENT.md

```diff
@@ -1,18 +1,63 @@
 # Development
```

Comment:
This could be extended to include perquisites for running the system locally and maybe something about the release process e.g. -Do updates, check everything is working, rebase, MR, merge, increase version, publish npm, test, MR, merge.